tasks/bundle-js.js

```diff
@@ -108,8 +108,8 @@ async function bundleJS(/** @type {JSEntry} */entry, {production}) {
                 cacheRoot: production ? null : `${fs.realpathSync(os.tmpdir())}/darkreader_typescript_cache`,
```

Comment:
So, there should be no more `production` property, but `debug` with inverted logic (if debug, use cache etc).